ouroboros-network/src/Ouroboros/Network/Diffusion.hs

```diff
@@ -350,17 +384,21 @@ runDataDiffusion tracers
         (
           case address of
                Left sd -> return sd
-               Right a -> Snocket.open sn (Snocket.addrFamily sn a)
+               Right addr -> do
+                 traceWith dtDiffusionInitializationTracer $ CreatingServerSocket addr
+                 Snocket.open sn (Snocket.addrFamily sn addr)
         )
         (Snocket.close sn) -- We close the socket here, even if it was provided for us.
         (\sd -> do
 
           case address of
                Left _ -> pure () -- If a socket was provided it should be ready to accept
                Right addr -> do
+                 traceWith dtDiffusionInitializationTracer $ ConfiguringServerSocket addr
```

Comment:
Similarly for this message, if `bind` errors diffusion stops => node stops. The error should be logged by the node.